io.github.mgerhardy.vengi.voxedit.yml

```diff
@@ -0,0 +1,58 @@
+app-id: io.github.mgerhardy.vengi.voxedit
```

Comment:
shouldn't this just be `io.github.mgerhardy.vengi`?

Reply:
There are more tools https://github.com/mgerhardy/vengi/pull/179#discussion_r963615244

Reply:
if they are build from the same source, they can be exported from the same flatpak. And have each their own desktop entry